test/app.js

```diff
@@ -0,0 +1,9 @@
+const tape = require('tape');
+const app = require('../src/app');
+
+
+tape('testing port', (t) => {
```

Comment:
we removed this too. This is why pulling an unmerged branch is bad.

Reply:
@FarahZaqout done :+1: